monadic-rx/shared/src/main/scala/mhtml/rx.scala

```diff
@@ -237,13 +237,36 @@ object Rx {
 }
 
 /** A smart variable that can be set manually. */
-class Var[A](initialValue: Option[A], register: Var[A] => Cancelable) extends Rx[A] {
+class Var[A](
+  initialValue: Option[A], register: Var[A] => Cancelable,
+  allowMultiMutations: Boolean = false
+) extends Rx[A] {
   // Last computed value, retained to be sent to new subscribers as they come in.
   private[mhtml] var cacheElem: Option[A] = initialValue
   // Current registration to the feeding `Rx`, canceled whenever nobody's listening.
   private[mhtml] var registration: Cancelable = Cancelable.empty
   // Mutable set of all currently subscribed functions, implementing with an `Array`.
   private[mhtml] val subscribers = buffer.empty[A => Unit]
+  // Keeps track of mutable call locations
+  private[mhtml] var mutationLocations: Set[Int] = Set()
```

Comment:
I would use the same mutable buffer used for `subscribers`, that might save putting all of `scala.collection.immutable.Set` in the js. Maybe it could be initialized at `null` to reduce the memory footprint in production?